Cargo.toml

```diff
@@ -42,3 +42,4 @@ exclude = [
 [dependencies]
 snafu-derive = { path = "snafu-derive", version = "0.2.3" }
 backtrace = { version = "0.3.0", optional = true }
+futures = "0.1.26"
```

Comment:
For the widest compatibility, we want to make this as permissive as possible, which is *likely* to be all the way back to just `0.1`.

Reply:
Could we also make `futures` an optional dep?

Reply:
@BurntSushi: That's the plan -- I just haven't formally added any associated features yet, so that simply hasn't been done yet. :)